src/components/EditorSideBar/EditorSideBar.tsx

```diff
@@ -0,0 +1,73 @@
+import EditorSideBarItem from "./EditorSideBarItem";
+import { Box, Flex } from "@chakra-ui/react";
+
+const EditorSideBar = () => {
+  const sideBarContent = [
+    {
+      id: "a",
```

Comment:
We don't need to create an new id for key, you can use the sideBarItemText as the key